modules/cert-manager-override-arguments.adoc

```diff
@@ -48,9 +49,10 @@ spec:
 ----
 <1> Provide a comma-separated list of `<host>:<port>` nameservers to query for the DNS-01 self check. For example, `--dns01-recursive-nameservers=1.1.1.1:53`.
 <2> Specify to only use recursive nameservers instead of checking the authoritative nameservers associated with that domain.
-<3> Provide a comma-separated list of `<host>:<port>` nameservers to query for the ACME HTTP01 self check. For example, `--acme-http01-solver-nameservers=1.1.1.1:53`.
+<3> Provide a comma-separated list of `<host>:<port>` nameservers to query for the Automated Certificate Management Environment (ACME) HTTP01 self check. For example, `--acme-http01-solver-nameservers=1.1.1.1:53`.
 <4> Specify to set the log level verbosity to determine the verbosity of log messages.
 <5> Specify the host and port for the metrics endpoint. The default value is `--metrics-listen-address=0.0.0.0:9402`.
+<6> You must use the `--issuer-ambient-credentials` argument when configuring an ACME issuer to solve DNS-01 challenges by using ambient credentials.
```

Comment:
In https://59702--docspreview.netlify.app/openshift-enterprise/latest/security/cert_manager_operator/cert-manager-operator-issuer-acme.html when you mention `ACME issuer`, some context or sections are talking about ClusterIssuer actually, so your wording `ACME issuer` seems to include Issuer and ClusterIssuer. But --issuer-ambient-credentials is only for Issuer. So, to be clear:
```suggestion
<6> You must use the `--issuer-ambient-credentials` argument when configuring an ACME Issuer to solve DNS-01 challenges by using ambient credentials.
```